components/status_dropdown/status_dropdown.jsx

```diff
@@ -17,12 +17,16 @@ import StatusAwayIcon from 'components/widgets/icons/status_away_icon';
 import StatusOnlineIcon from 'components/widgets/icons/status_online_icon';
 import StatusDndIcon from 'components/widgets/icons/status_dnd_icon';
 import StatusOfflineIcon from 'components/widgets/icons/status_offline_icon';
+import DndCustomTimePicker from 'components/dnd_custom_time_picker_modal';
+import {getCurrentDateTimeForTimezone} from 'utils/timezone';
 
 export default class StatusDropdown extends React.PureComponent {
     static propTypes = {
         style: PropTypes.object,
         status: PropTypes.string,
         userId: PropTypes.string.isRequired,
+        userTimezone: PropTypes.object,
+        enableTimezone: PropTypes.bool,
```

Comment:
I am not entirely sure if this wording is the best we can use for that. Since it is a boolean it should probably be something like this: `isTimezoneEnabled`

WDYT @jespino ?

Reply:
Yes, I agree with that, `enableTimezone` looks like an action. I think isTimezoneEnable is clearer.